### PR TITLE
Encryption support

### PR DESCRIPTION
**Pull request by glennbech**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -62,6 +62,11 @@ variable "snapshot_identifier" {
   default     = ""
 }
 
+variable "storage_encrypted" {
+  description = "Specifies whether the DB instance is encrypted."
```
Comment (itsdalmo) [inline variables.tf]: "DB cluster" 😄

**Comment by mikael-lindstrom** [comment]
Question, should we also expose `kms_key_id`?

File: variables.tf (hunk 1 above)
Comment (itsdalmo) [reply variables.tf]: Btw, will this work without also exposing `kms_key_id`? Do they have a default encryption key? Have you tried it?

**Comment by glennbech** [comment]
@mikael-lindstrom I can do that while I'm at it.

**Review by mikael-lindstrom — APPROVED** [review]
LGTM

**Pull request merged**